Sources/SourceKitLSP/Swift/SemanticTokens.swift

```diff
@@ -35,7 +35,9 @@ extension SwiftLanguageService {
     guard let skTokens: SKDResponseArray = dict[keys.semanticTokens] else {
       return nil
     }
-    return SyntaxHighlightingTokenParser(sourcekitd: sourcekitd).parseTokens(skTokens, in: snapshot)
+
+    let parsedTokens = SyntaxHighlightingTokenParser(sourcekitd: sourcekitd).parseTokens(skTokens, in: snapshot)
+    return SyntaxHighlightingTokens(members: parsedTokens)
```

Comment:
I think we should make `parseTokens` already return `SyntaxHighlightingTokens` instead of `[SyntaxHighlightingToken]`. Then we don’t need to construct `SyntaxHighlightingTokens` here.